### DISCUSSION TITLE
Save image : filename prefix --> how to generate a date after the value (text add tokens?)

### DISCUSSION THREAD

**Discussion by Rojinski** [WASasquatch/was-node-suite-comfyui#149] [Q&A] [answered]
Hello everyone,

I've installed the "was node suite" because it can generate automatically a date when you save an image by using a node "text add tokens". I'm an ultra newbie in using nodes and Comfy UI (and I'm a 58 yo sound engineer / composer - it doesn't fix things). What I'd like to do is when I save my image, it automatically name my file like this "MyName_date_0001" (example).  I've tried string like this in the value --> [time(%Y-%m-%d)]_[time(%H-%M-%S)] but it just saves that name and add a number to it "0001, 0002, etc"... 

I work on this since the last 4 hours (google - github -youtube) but, well, it just gave me suicide ideas... LOL (kidding). Could someone help me, please?  

Friendly,

Rojinski

**Comment by WASasquatch** [comment]
I may not be parsing date codes in the filename prefix and only the path. I will check. Sounds like me to forget that though. I will check here shortly.

**Comment by WASasquatch** [comment]
I think I fixed it. It looks like filename prefix parsing came after it was used to define the filename.

![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/1151589/9a3b444c-5de4-4583-8b80-994dfd327e7e)

![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/1151589/d2aae518-efa3-4096-9317-5d58a417604c)

**Reply by Rojinski** [reply]
Awww... Thank you for your fast answer. Very grateful. And how  can I do to  change the path? Do i have to open the node "save image" and do a thing like [time(%Y-%m-%d)]_[time(%H-%M-%S)]/my_name? As I said, I'm a complete newbie with ComfyUI... I feel like a child facing a SSL Recording console... I'm so sorry to disturb you with such stupid questions....  I don't have all the options you have on your picture.... :/ 
![Screenshot 2023-08-01 at 18-25-56 ComfyUI](https://github.com/WASasquatch/was-node-suite-comfyui/assets/122015375/edc7b64e-52d3-41cc-89e8-90b9bbfeba1c)

**Comment by WASasquatch** [comment]
You can do the path in the output_path field. Currently by default it's
creating a folder with the date. But you could do like:
MyAwesomeProject/[time(%Y-%m-%d)]

Which would make two folders: /MyAwesomeProject/2023-08-01/

**Reply by Rojinski** [reply]
I did it.... but it just give me a folder named "[time(%Y-%m-%d)]"... How can I have the node you show on your picture above? What may I choose as a node? I'm really lost... Sorry. And thank you again for your help.

Edit: here's my workflow... Maybe you will understand it better...

[https://www.mediafire.com/file/v7zfrbibqvyvsku/Roji-SDXL-Best.json/file](url)

**Reply by WASasquatch** [reply, marked as answer]
You're not using my Save Image node, that's the base vanilla ComfyUI save image node. My save image node is under `WAS Suite -> IO -> Save Image`

**Reply by Rojinski** [reply]
OMG!!! Thanks a lot! I didn't check the IO folder to be honnest. Was looking in images and text folders.... Thank you so much!!!!!!!!!

**Reply by tusharbhutt** [reply]
Don't feel bad, I did the same thing :)
